GalaxyZooTouchTable/GalaxyZooTouchTable/Models/TableSubject.cs

```diff
@@ -5,21 +5,71 @@ namespace GalaxyZooTouchTable.Models
 {
     public class TableSubject
     {
-        public string X { get; set; }
-        public string Y { get; set; }
+        public int X { get; set; }
+        public int Y { get; set; }
         public double RA { get; set; }
         public double DEC { get; set; }
         public string SubjectLocation { get; set; }
         private double PlateScale { get; set; } = 0.4;
         public Subject Subject { get; set; }
 
-        public TableSubject(Subject subject, double tableRA, double tableDec)
+        public TableSubject(Subject subject, double TableRA, double TableDEC)
         {
             RA = Convert.ToDouble(subject.Metadata.ra);
             DEC = Convert.ToDouble(subject.Metadata.dec);
-            var test = (RA - tableRA) * (PlateScale / Math.Cos(DEC));
+            var test = (RA - TableRA) * (PlateScale / Math.Cos(DEC));
             Subject = subject;
             SubjectLocation = subject.GetSubjectLocation();
+            EQConvert(TableRA, TableDEC);
+        }
+
+        private void EQConvert(double TableRA, double TableDEC)
+        {
+            double PI = 3.1415926535897932384626433832795;
+            double TwoPI = 6.283185307179586476925286766559;
+            double PI2 = 1.5707963267948966192313216916398;
+            double Rads = 0.01745329251994329576923690768489;
+
+            double Az, Alt, tmpval, nz, az2, tx, ty, xyscale;
+
+            double DisplayFOV = 1.4;
+            double DisplayROT = 0.0;
```

Comment:
I know there is a lot of math here and the `FOV` and `ROT` probably stand for some math stuff, but I think it would make sense to either rename or add a large comment explaining what these values mean.